src/C2lcURLParams.js

```diff
@@ -18,4 +20,12 @@ export default class C2lcURLParams {
     getCharacterState() {
         return this.urlSearchParams.get('c');
     }
+
+    getTheme(): ThemeNames {
```

Comment:
I think I'd prefer to keep the responsibility of C2lcURLParams class to extracting strings from the URL. If we parse the theme here, we have an asymmetry with regards to program and character state as we don't parse these here. Let's keep this class limited to returning strings. And do the parsing elsewhere (in App).